src/components/recursive_tree/recursive_tree.tsx

```diff
@@ -71,8 +75,7 @@ export type EuiRecursiveTreeProps = Omit<
 
 export class EuiRecursiveTree extends Component<
   EuiRecursiveTreeProps,
-  EuiRecursiveTreeState,
-  CommonTreeProps
```

Comment:
Deleted this because `EuiRecursiveTreeProps` is a superset so that should be all you need... Though I could be wrong...

Reply:
As the third argument to `Component<x,y,z>` it was effectively doing nothing anyway 👍